exercises/stack_02.js

```diff
@@ -0,0 +1,167 @@
+//Stack.push: one
+window.Collections = Object.create(null);
+
+(function(root) {
```

Comment:
Throughout this file, you keep re-using the module construct an IFFE to implement each method. But, you don't need to do that. You can wrap *the whole module* in one IFFE. See stack_01.js for an example of this.